assets/src/scripts/components/hydrograph/layout.js

```diff
@@ -70,7 +70,7 @@ export const getLayout = memoize(kind => createSelector(
                 ...margin,
                 top: kind === 'BRUSH' ? 0 : margin.top,
                 left: margin.left + approxLabelLength,
-                right: margin.right + isTemperatureParameter ? approxLabelLength : 0
+                right: margin.right + approxLabelLength
```

Comment:
Please restore the line 73 that is on master. This should be 
```right: margin.right + isTemperatureParameter ? approxLabelLength : 0```

Reply:
I was able to restore this line, the Tests no longer are erroring with it.  Strange.